### PR TITLE
Make module work in runtimes without  module

### PR DESCRIPTION
**Pull request by steelbrain**
cc @Arcanemagus

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.5.1
+
+*   Fix for runtimes who don't have `atom` module
```
Comment (Arcanemagus) [inline CHANGELOG.md]: `who` -> `which`

Comment (Arcanemagus) [inline CHANGELOG.md]: Also "have **the** `atom` module available"

Comment (steelbrain) [inline CHANGELOG.md]: Done

**Comment by Arcanemagus** [comment]
Looks like CircleCI is currently experiencing high queue times: http://status.circleci.com/incidents/ddpngcbmmhdd

Since Travis-CI also tests Linux builds you have my go to merge at will.

**Pull request merged**